src/Symfony/Component/Serializer/Attribute/SerializedPath.php

```diff
@@ -18,27 +18,43 @@
 /**
  * @author Tobias Bönner <tobi@boenner.family>
  */
-#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY | \Attribute::IS_REPEATABLE)]
 class SerializedPath
 {
     private PropertyPath $serializedPath;
 
+    private array $groups;
+
     /**
-     * @param string $serializedPath A path using a valid PropertyAccess syntax where the value is stored in a normalized representation
+     * @param string          $serializedPath A path using a valid PropertyAccess syntax where the value is stored in a normalized representation
+     * @param string|string[] $groups         The groups to use when serializing or deserializing
      */
-    public function __construct(string $serializedPath)
+    public function __construct(string $serializedPath, string|array $groups = [])
```

Comment:
```suggestion
    public function __construct(string $serializedPath, string|array $groups = ['*'])
```
And then we can check that `$groups` is not empty
(same for `SerializedName`)

Reply:
What should we do in case `$groups` is empty? Throw exception or fall back to `['*']`?
(BTW, I've implemented second option).